x/crosschain/keeper/msg_server_vote_gas_price_test.go

```diff
@@ -262,4 +262,62 @@ func TestMsgServer_VoteGasPrice(t *testing.T) {
 		assert.Equal(t, []uint64{3}, gp.Prices)
 		assert.Equal(t, []uint64{2}, gp.PriorityFees)
 	})
+
+	t.Run("tolerates lack of priorityFee", func(t *testing.T) {
```

Comment:
Can we also test when an existing creating update the price?

Reply:
Do you mean this one? https://github.com/zeta-chain/node/blob/1608132eb91747e77eaecc120bf70128a834a069/x/crosschain/keeper/msg_server_vote_gas_price_test.go#L211

Reply:
This one didn't catch the bug because I think the first call of `VoteGasPrice` will still push a zeroed PriorityFees, I meant a test where for a creator, the priority fee is explicitly not set in the structure.

Reply:
okay, got it.